channels/models.py

```diff
@@ -0,0 +1,54 @@
+"""Channels models"""
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from django.db import models
+
+from open_discussions.utils import now_in_utc
+
+
+class RedditRefreshToken(models.Model):
+    """
+    Tracks the refresh token for a given user
+    """
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    token_value = models.CharField(null=True, max_length=255)
+
+    def __str__(self):
+        return self.token_value
+
+
+class RedditAccessToken(models.Model):
+    """
+    Tracks the access tokens for a given user
+
+    We generate a new one whenever the latest one expires and then clean up stale ones in a cron
+    """
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    token_value = models.CharField(null=True, max_length=255)
+    token_expires_at = models.DateTimeField(null=True)
+
+    @property
+    def expires_in(self):
+        """Returns the number of seconds until the token expires"""
+        return (now_in_utc - self.token_expires_at).total_seconds()
```

Comment:
`now_in_utc()`

Reply:
Also, what happens if `token_expires_at` is `None`? Are we assuming that's a rare case and just want to let it fail?

Reply:
Heh, well, seeing as I just noticed this property is no longer used, how about we split the difference and remove it?